ci/ubuntu-install-dependencies.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -euo pipefail
+IFS=$'\n\t'
+
+sudo apt-get update
+
+# only for arci-gamepad-gilrs
+sudo apt-get install -y libudev-dev libasound2-dev
```

Comment:
libasound2-dev is for arci-speak-audio (The original comment was already wrong).
Could you update the comment?

Reply:
Updated the comment.